coretex/entities/sample/network_sample.py

```diff
@@ -112,6 +114,31 @@ def _createSample(
 
         return cls.decode(response.json)
 
+    def modifiedSinceLastDownload(self) -> bool:
+        """
+            Checking if sample has been modified since last download, if the sample is already
+            stored locally
+
+            Returns
+            -------
+            bool -> False if sample has not changed since last download, True otherwise
+        """
+
+        if not self.zipPath.exists():
+            return False
```

Comment:
Should an exception be raised if the file does not exist?

Reply:
No. The file will only exist if was already downloaded before, so in case this is the first download it is expected for it not to exist

Reply:
Let's update this function to raise an exception if the sample file does not exist